mmcv/ops/csrc/pytorch/npu/roi_pool_npu.cpp

```diff
@@ -0,0 +1,34 @@
+#include "pytorch_npu_helper.hpp"
+
+using namespace NPU_NAME_SPACE;
+using namespace std;
+
+Tensor roi_pool_forward_npu(Tensor input, Tensor rois, Tensor output,
```

Comment:
Should `backward` also be supported?

Reply:
backward op is still under development and will be supported later

Reply:
The return type is different from `roi_pool_forward_impl`